crates/orderbook/src/arguments.rs

```diff
@@ -144,9 +144,11 @@ pub struct Arguments {
     #[clap(long, env, default_value = "24")]
     pub solvable_orders_max_update_age_blocks: u64,
 
+    /// Note that fill or kill liquidity limit orders are always allowed.
```

Comment:
Did you mean this?
```suggestion
    /// Note that fill or kill liquidity orders are always allowed.
```

Reply:
The original text is what I meant. But I think what you are getting at, is that the distinction of "liquidity and limit" is pointless because we have three distinct order types: market, limit, liquidity and liquidity don't count as the other two so I don't need to mention that here.